### PR TITLE
[PT] Translate Behaviours

### PR DESCRIPTION
**Pull request by vicdda**
Hey, i have done the translation of Behaviours, please review.
Thanks !

### CODE REVIEW

**Comment by doomspork** [comment]
Thank you @vicdda!  It looks like there's a conflict with one of the files, could you take a peek at that?

@pragmaticivan could you have a look at this when you're free?  Thanks!

**Comment by vicdda** [comment]
Hey @doomspork, done !
Please see if everything is correct.
Sorry and Thanks !

File: _config.yml [hunk 1]
```diff
@@ -218,10 +219,4 @@ version_messages:
...
     minor: "Einige kleinere Änderungen wurden der originalen Lektion seit der letzten Aktualisierung hinzugefügt."
     patch: "Einige Patches wurden der originalen Lektion seit der letzten Aktualisierung hinzugefügt."
-    none: "Diese Übersetzung ist aktuell."
-  es:
-    outdated: "Algunos contenidos de esta tradución pueden estar desactualizados"
-    major: "Varios cambios mayores fueron aplicados a la lección original desde la última actualización."
-    minor: "Varios cambios menores fueron aplicados a la lección original desde la última actualización."
-    patch: "Varios parches fueron aplicados a la lección original desde la última actualización"
-    none: "Esta traducción está actualizada."
+    none: "Diese Übersetzung ist aktuell."
```
Comment (pragmaticivan) [inline _config.yml]: I think you did some unnecessary removal here.

**Comment by doomspork** [comment]
@vicdda could you rebase your branch so we could merge these changes?  Thank you again!

**Comment by anderkonzen** [comment]
@doomspork @vicdda it seems the Behaviors' translation for Portuguese was merged via another PR (#1226). I believe we can close this one?

**Comment by doomspork** [comment]
Thanks for bringing this to my attention @anderkonzen! 😁  

@anderkonzen would you be interested in joining the PT translation team we have?

**Comment by anderkonzen** [comment]
Hi @doomspork! About your question, yes, I'm interested in joining the team. That would be awesome!

**Pull request closed without merging**